CHANGELOG.md

```diff
@@ -327,7 +270,7 @@ The extension takes care of the necessary configuration required to authenticate
 
 The future of the [JMX metrics](./jmx-metrics/README.md) component,
 built on top of the
-[JMX metrics](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics#jmx-metric-insight)
+[JMX metrics](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/README.md#jmx-metric-insight)
```

Comment:
the original link is a bit better in this case (and isn't broken), how messy would the regex get if we wanted to accommodate it?